server/app/services/export/CsvExporterService.java

```diff
@@ -129,32 +129,23 @@ private CsvExportConfig generateCsvConfig(
       ImmutableMap<Long, ProgramDefinition> programDefinitionsForAllVersions,
       boolean showEligibilityColumn)
       throws ProgramNotFoundException {
-    Map<Path, AnswerData> answerMap = new HashMap<>();
+    Map<Path, ApplicantQuestion> uniqueQuestions = new HashMap<>();
 
     applications.stream()
         .flatMap(
             app ->
                 applicantService
                     .getReadOnlyApplicantProgramService(
                         app, programDefinitionsForAllVersions.get(app.getProgram().id))
-                    .getSummaryDataAllQuestions()
-                    .stream())
-        .forEach(
-            data -> {
-              answerMap.putIfAbsent(data.contextualizedPath(), data);
-            });
-
-    // Get the list of all answers, sorted by block ID, then question index, and finally
-    // contextualized path in string form.
-    ImmutableList<AnswerData> answers =
-        answerMap.values().stream()
-            .sorted(
-                Comparator.comparing(AnswerData::blockId)
-                    .thenComparing(AnswerData::questionIndex)
-                    .thenComparing(answerData -> answerData.contextualizedPath().toString()))
+                    .getAllQuestions())
+        .forEach(aq -> uniqueQuestions.putIfAbsent(aq.getContextualizedPath(), aq));
+
+    ImmutableList<ApplicantQuestion> exemplarQuestions =
```

Comment:
**nitpick:** I'm good with keeping it as is, but thoughts on a name like sortedQuestions?